com.unity.shadergraph/Editor/Data/Nodes/Procedural/Noise/VoronoiNode.cs

```diff
@@ -58,7 +58,7 @@ public override void GenerateNodeFunction(FunctionRegistry registry, GenerationM
 inline $precision2 Unity_Voronoi_RandomVector_$precision ($precision2 UV, $precision offset)
 {
     $precision2x2 m = $precision2x2(15.27, 47.63, 99.41, 89.98);
-    UV = frac(sin(mul(UV, m)) * 46839.32);
```

Comment:
can we simply change this number to be `46839` to try and maintain more consistent behavior?

Reply:
no, removing just the fractional part does not fix the issue